src/services/recim.ts

```diff
@@ -1,3 +1,31 @@
+import { DateTime } from 'luxon';
 import http from './http';
 
 // all Gordon Rec-IM http requests
+type League = {
+  ID: number;
+  Name: string;
+  RegistrationStart: DateTime;
```

Comment:
I don't think this does what you think it does. This 'DateTime' is completely unrelated to C#'s DateTime type. Also, because you're serializing/deserializing JSON with this type, it will just be a string in transit. That means that you would have to manually convert the string date into a Luxon DateTime, which is not what you are doing.

Reply:
ahhh, i see okay. I made an assumption that it worked because it did not throw me errors when running